src/core/sprites/webgl/SpriteRenderer.js

```diff
@@ -351,7 +351,7 @@ export default class SpriteRenderer extends ObjectRenderer
             uint32View[index + 12] = uvs[2];
             uint32View[index + 17] = uvs[3];
 
-            uint32View[index + 3] = uint32View[index + 8] = uint32View[index + 13] = uint32View[index + 18] = sprite._tintRGB + (sprite.worldAlpha * 255 << 24);
+            uint32View[index + 3] = uint32View[index + 8] = uint32View[index + 13] = uint32View[index + 18] = sprite.tintRgbInt + (sprite.worldAlpha * 255 << 24);
```

Comment:
`_tintRGB` was calculated only once here when changing `tint`. Your addition will calculate `tintRgbInt` everytime. Not sure this is an improvement.